### ISSUE TITLE
Compute the "local gradients"?

### ISSUE THREAD

**Issue by andrewliao11** [tensorflow/tensorflow#4506]
Hi all, 
I wonder if the tensorflow can do the backprop given the gradient from the loss?
eg.
based on the chain rule, for y=f(x)
dloss/dx = dloss/dy * dy/dx
Can I do the backprop given the dloss/dy?

**Comment by vladfi1** [comment]
I believe `tf.gradients(y, x, grad_ys=<dloss/dy>)` does what you want.

**Comment by andrewliao11** [comment]
thx! solved
